src/coherence/nslookup.py

```diff
@@ -0,0 +1,293 @@
+import asyncio
```

Comment:
Missing copyright.

Reply:
If this module and the classes within are to be public, ensure a) it's included in the documentation list and the code is documented.

Reply:
This module is not public. Copyright added.